tests/beaker_tests/cisco_bgp/test_bgp_non_defaults.rb

```diff
@@ -146,6 +148,8 @@
   expected_values_vrf1['suppress_fib_pending']                   = 'false'
   expected_values_vrf1['timer_bestpath_limit']                   = '255'
   expected_values_vrf1['timer_bestpath_limit_always']            = 'true'
+else
+  expected_values_vrf1['nsr']                                    = 'false'
```

Comment:
Seems a bit unnecessary since below you are specifically deleting this key - why bother setting it if you're always going to delete it?